### ISSUE TITLE
The database files are unavailable for downloading

### ISSUE THREAD

**Issue by andrewgull** [dutilh/CAT#34]
Dear Bastiaan,
the command `wget tbb.bio.uu.nl/bastiaan/CAT_prepare/CAT_prepare_20190719.tar.gz` is not working because tbb.bio.uu.nl is not available

**Comment by Finesim97** [comment]
Hi, just checked, they are back online

**Comment by andrewgull** [comment]
Well, my attempt has finished like this

```
wget tbb.bio.uu.nl/bastiaan/CAT_prepare/CAT_prepare_20190719.tar.gz
--2020-03-05 13:04:12--  http://tbb.bio.uu.nl/bastiaan/CAT_prepare/CAT_prepare_20190719.tar.gz
Resolving tbb.bio.uu.nl (tbb.bio.uu.nl)... failed: Name or service not known.
wget: unable to resolve host address ‘tbb.bio.uu.nl’
```

**Comment by Finesim97** [comment]
Its gone again, they seem to have problems/are changing their DNS configuration. My download is still running. wget luckily outputs the IP, try this:
```
wget http://131.211.52.141/bastiaan/CAT_prepare/CAT_prepare_20190719.tar.gz
```

**Comment by andrewgull** [comment]
Thanks!
Now it works

**Comment by bastiaanvonmeijenfeldt** [comment]
Hi all,

Indeed we have had some trouble with the DNS. Thanks again @Finesim97 for troubleshooting! It seems to be working on and off, if the issue remains we'll try to figure out with IT what's going on. On a different note, I've updated the database files on our servers (2020-03-04), so feel free to wget those to get the latest nr database!

`wget tbb.bio.uu.nl/bastiaan/CAT_prepare/CAT_prepare_20200304.tar.gz`

or

`wget 131.211.52.141/bastiaan/CAT_prepare/CAT_prepare_20200304.tar.gz`

Closing this issue.
